contracts/test/integration/IntegrationTestCompoundPCVDeposits.t.sol

```diff
@@ -43,49 +43,18 @@ contract IntegrationTestCompoundPCVDeposits is TimelockSimulation, vip9 {
     uint256 public usdcBalance;
 
     function setUp() public {
-        daiBalance = dai.balanceOf(MainnetAddresses.CDAI);
-        vm.prank(MainnetAddresses.CDAI);
-        dai.transfer(address(daiDeposit), daiBalance);
-        daiDeposit.deposit();
+        daiBalance = daiDeposit.balance();
+
+        feiBalance = feiDeposit.balance();
 
-        feiBalance = fei.balanceOf(MainnetAddresses.CFEI);
-        vm.prank(MainnetAddresses.CFEI);
-        fei.transfer(address(feiDeposit), feiBalance);
-        feiDeposit.deposit();
+        usdcBalance = usdcDeposit.balance();
 
-        usdcBalance = usdc.balanceOf(MainnetAddresses.CUSDC);
         vm.prank(MainnetAddresses.CUSDC);
         usdc.transfer(address(usdcDeposit), usdcBalance);
         usdcDeposit.deposit();
```

Comment:
how come we maintain this for usdc but not for dai?

Reply:
because USDC has no assets in it to start, but fei and dai deposits do.